test_package/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ if(MSVC)
   string(REGEX REPLACE "/RTC(su|[1su])" "" CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
```

Comment:
Sure, no problem.

Reply:
I'm not sure what this line does. Can we replace CMAKE_CXX_FLAGS_DEBUG simply with CMAKE_C_FLAGS_DEBUG and it will be fine?
Is this relevant for C only?

Reply:
this code disables runtime checks for debug builds, removing /RTC flags (all its flavors) from the compiler command line
but I doubt it's needed at all, since it passes without it, so I suggest to just remove these line